tests/console/command_not_found.pm

```diff
@@ -30,19 +31,21 @@ sub run {
     my $not_installed_pkg = (is_sle && sle_version_at_least '15') ? 'wireshark' : 'xosview';
     my $cnf_cmd           = "echo \"\$(cnf $not_installed_pkg 2>&1 | tee /dev/stderr)\" | grep -q \"zypper install $not_installed_pkg\"";
     my $assert            = 1;
```

Comment:
so this variable can be removed if not use anymore

Reply:
Nice catch, thanks!